RecoMET/METPUSubtraction/python/mvaPFMET_leptons_cfi.py

```diff
@@ -4,100 +4,98 @@
+                     "&& hcalOverEcal                                   < 0.15" +
+                     "&& std::abs(1./superCluster.energy - 1./p)             < 0.05)"+
+                     "|| (std::abs(eta)  > 1.566 "+
+                     "&& std::abs(deltaEtaSuperClusterTrackAtVtx)            < 0.009"+
+                     "&& std::abs(deltaPhiSuperClusterTrackAtVtx)            < 0.10" +
+                     "&& sigmaIetaIeta                                  < 0.03" +
+                     "&& hcalOverEcal                                   < 0.10" +
+                     "&& std::abs(1./superCluster.energy - 1./p)             < 0.05))" 
+                     ),
+    filter = cms.bool(False)
+)
 
 from RecoJets.Configuration.RecoPFJets_cff import kt6PFJets as dummy
 kt6PFJetsForRhoComputationVoronoiMet = dummy.clone(
-        doRhoFastjet = True,
-        voronoiRfact = 0.9
-        )
+    doRhoFastjet = True,
+    voronoiRfact = 0.9
+)
 
 from RecoTauTag.RecoTau.PFRecoTauDiscriminationByHPSSelection_cfi import hpsSelectionDiscriminator
 hpsPFTauDiscriminationByDecayModeFinding = hpsSelectionDiscriminator.clone(
-        PFTauProducer = 'hpsPFTauProducer'
-            )
+    PFTauProducer = 'hpsPFTauProducer'
+)
 
 from RecoTauTag.RecoTau.TauDiscriminatorTools import requireLeadTrack
 # Define decay mode prediscriminant
 requireDecayMode = cms.PSet(
-        BooleanOperator = cms.string("and"),
-            decayMode = cms.PSet(
-            Producer = cms.InputTag('hpsPFTauDiscriminationByDecayModeFinding'),
-                    cut = cms.double(0.5)
-                )
-        )
+    BooleanOperator = cms.string("and"),
+    decayMode = cms.PSet(
+        Producer = cms.InputTag('hpsPFTauDiscriminationByDecayModeFinding'),
+        cut = cms.double(0.5)
+    )
+)
 
 from RecoTauTag.Configuration.HPSPFTaus_cff import hpsPFTauDiscriminationByLooseCombinedIsolationDBSumPtCorr3Hits
+import RecoTauTag.RecoTau.pfRecoTauDiscriminationAgainstMuon2_cfi as _mod
 
-hpsPFTauDiscriminationAgainstMuon2 = cms.EDProducer("PFRecoTauDiscriminationAgainstMuon2",
-                                                         PFTauProducer = cms.InputTag('hpsPFTauProducer'),
-                                                         Prediscriminants = requireDecayMode.clone(),
-                                                         discriminatorOption = cms.string('loose'), # available options are: 'loose', 'medium', 'tight'
-                                                         HoPMin = cms.double(0.2)
-                                                     )
-
+PFTauDiscriminationAgainstMuon2 = _mod.pfRecoTauDiscriminationAgainstMuon2.clone(
+    PFTauProducer = 'hpsPFTauProducer',
+    Prediscriminants = requireDecayMode.clone(),
+    discriminatorOption = 'loose', # available options are: 'loose', 'medium', 'tight'
+)
 
 hpsPFTauDiscriminationByMVAIsolation = cms.EDProducer(
     "PFRecoTauDiscriminationByMVAIsolation",
```

Comment:
I can't find this plugin, while I see `PFRecoTauDiscriminationByMVAIsolationRun2` instead
@mbluj @swozniewski is it just me, or this configuration is outdated by now?